ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/misc.py

```diff
@@ -372,14 +372,14 @@ def new_network_services_bgp_vrf_config(self: SharedUtils) -> bool:
 
     @cached_property
     def ipv4_acls(self: SharedUtils) -> list:
```

Comment:
It is not clear to me why you are changing this model to a dict. Make sure all references to it has been adjusted accordingly.
```suggestion
    def ipv4_acls(self: SharedUtils) -> dict:
```

Reply:
We always are doing lookup on this, and ordering isn't relevant, better to just have it as dict.